modules/gitops-openshift-go-terminology.adoc

```diff
@@ -0,0 +1,47 @@
+Refresh:: The process to compare the latest code in Git with the live state and figure out the difference.
+AppProject:: A resource object and CRD representing a logical grouping of applications within a project, which governs where and how an application might manage resources. 
+Argo CD instance:: A copy or an instantiation that encapsulates all of the stateful aspects of a running Argo CD.
+Instance scopes:: Modes that determine how you can want to operate an Argo CD instance. The available modes are cluster-scoped instance and namespace-scoped instance.
+Namespace-scoped instance (Application delivery instance):: A mode wherein Argo CD is configured to only manage resources in certain namespaces on a cluster and use them for application delivery.
+Cluster-scoped instance:: A mode wherein Argo CD is configured to manage every single resource on the cluster including certain cluster-specific resources such as platform Operators, optional OLM Operators, and secrets.
+Default cluster-scoped instance:: A default instance that a {gitops-title} Operator instantiates immediately after its installation, in the `openshift-gitops` namespace.
+ApplicationSet (Argo CD application set):: A resource object and a CRD that automatically generates Argo CD applications based on the contents of an `ApplicationSet` CR. Cluster administrators use this CRD to define a single `ApplicationSet` CR to generate and update multiple corresponding Argo CD `Application` CRs.
+Argo CD API server:: A gRPC/REST server that exposes the API consumed by the Web UI, CLI, continuous integration (CI), and continuous deployment (CD) systems. 
+Ardo CD server (Argo CD-server):: A server that serves the API and UI for Argo CD.
+Argo CD repository server (Argo CD-repo-server):: A server that reads from Git repositories that hold the application manifests, generates manifests, and executes custom configuration management tooling. 
+Argo CD resource (`ArgoCD` CR):: A CRD that describes the desired state for a given Argo CD cluster that allows you to configure the components which make up an Argo CD cluster.
+Application Controller (Argo CD Application Controller):: A controller that continuously watches the Git repo for changes, monitors running applications, compares the current, live state against the desired target state, and deploys new changes. Examples include Argo CD Application Controller  detecting OutOfSync application state and optionally taking corrective action.
+ApplicationSet Controller (Argo CD ApplicationSet Controller):: A custom Kubernetes controller that installs alongside Argo CD and adds support for a custom `ApplicationSet` CRD.  This controller automatically generates Argo CD Applications based on the contents of an `ApplicationSet` CR and manages them, across multiple clusters declaratively, from a single mono-repository. The Argo CD ApplicationSet Controller is shipped as part of the {gitops-title} Operator and Argo CD in all currently-supported versions. 
```

Comment:
> ApplicationSet Controller (Argo CD ApplicationSet Controller):

Recommend changing first sentence:
A custom Kubernetes controller that installs alongside Argo CD and adds support for a custom `ApplicationSet` CRD.
->
A custom Kubernetes controller that **_exists within Argo CD_** and adds support for a custom `ApplicationSet` CRD.

(As mentioned in the final sentence, at this point ApplicationSet controller is fully integrated with Argo CD, so I would argue ApplicationSet is 'within' Argo CD, rather than alongside it)

---

IMHO this sentence can be removed, as I think it is not longer required to indicate it is now fully integrated with Argo CD.
`The Argo CD ApplicationSet Controller is shipped as part of the {gitops-title} Operator and Argo CD in all currently-supported versions.`